test/strategy.coffee

```diff
@@ -98,6 +100,9 @@ describe 'EVE Online OAuth Strategy', ->
     it 'should use the tokenURL property provided', ->
       @constructorOptions.should.have.property('tokenURL').equal @customTokenURL
 
+    it 'should use the scopes property provided', ->

```

Comment:
The test should be part of a different context, since not every context will have scopes.  For example:
```coffeescript
describe 'when created with scopes' ->
    beforeEach ->
      @scopes = 'some scopes'
      @strategy = new EveOnlineStrategy(
        clientID: @clientID
        clientSecret: @clientSecret
        callbackURL: @callbackURL
        scopes: @scopes
        @verify)
      @constructorOptions = @strategy.parentConstructor.args[0][0]

    it 'should use the scopes property provided', ->
      @constructorOptions.should.have.property('scopes').equal @scopes
```